buildenv

```diff
@@ -28,29 +21,16 @@ zopen_check_results()
   echo "expectedFailures:0"
 }
 
-zopen_test()
-{
-  make test
-}
-
-zopen_install()
-{
-  go install
-}
-
 zopen_append_to_env()
 {
   # echo envars outside of PATH, MANPATH, LIBPATH
 }
 
 zopen_append_to_setup()
 {
-   direnv_eval="$(eval "$(direnv hook bash)")"
-   echo "$direnv_eval" >> "~/.bashrc"
-   exec bash -l
```

Comment:
This is needed to activate direnv.
Should this instead be shown as post-install user actions? How would I do that?